cg/models/report/validators.py

```diff
@@ -1,100 +1,77 @@
 from cg.models.orders.constants import OrderType
 
 LOG = logging.getLogger(__name__)
 
 
-def validate_empty_field(value: Union[int, str]) -> str:
-    """Formats an empty value to be included in the report as N/A."""
+def get_report_string(value: Any) -> str:
+    """Return report adapted string."""
     return str(value) if value else NA_FIELD
 
 
-def validate_boolean(value: Union[bool, str]) -> str:
-    """Formats a boolean value for the delivery report."""
-    if isinstance(value, bool) or value:
-        if str(value) == "True":
-            return YES_FIELD
-        if str(value) == "False":
-            return NO_FIELD
+def get_boolean_as_string(value: Optional[bool]) -> str:
+    """Return delivery report adapted string representation of a boolean."""
+    if isinstance(value, bool):
+        return YES_FIELD if value else NO_FIELD
     return NA_FIELD
 
 
-def validate_float(value: Union[float, str]) -> str:
-    """Returns a processed float value."""
+def get_float_as_string(value: Optional[float]) -> str:
+    """Return string representation of a float value."""
     return str(round(float(value), PRECISION)) if value or isinstance(value, float) else NA_FIELD
 
 
-def validate_percentage(value: Union[float, str]) -> str:
-    """Returns a processed float value as a percentage."""
-    return validate_float(float(value) * 100) if value else NA_FIELD
+def get_float_as_percentage(value: Optional[float]) -> str:
+    """Return string percentage representation of a float value."""
+    return get_float_as_string(value * 100) if value or isinstance(value, float) else NA_FIELD
 
 
-def validate_date(date: datetime) -> str:
-    """Returns the date part (year, month, day) from a datetime object."""
+def get_date_as_string(date: Optional[datetime]) -> str:
+    """Return the date string representation (year, month, day) of a datetime object."""
     return str(date.date()) if date else NA_FIELD
 
 
-def validate_list(value: list) -> str:
-    """Formats a list elements as comma separated individual values."""
-    return validate_empty_field(
-        ", ".join(validate_empty_field(v) for v in value) if value else NA_FIELD
-    )
+def get_list_as_string(value: Optional[list[str]]) -> str:
+    """Return list elements as comma separated individual string values."""
+    return ", ".join(v for v in value) if value else NA_FIELD
 
 
-def validate_path(file_path: str) -> str:
-    """Returns the name of a specific file."""
+def get_path_as_string(file_path: Optional[str]) -> str:
+    """Return a report validated file name."""
     return Path(file_path).name if file_path and Path(file_path).is_file() else NA_FIELD
 
 
-def validate_gender(value: str) -> str:
-    """Formats the provided gender."""
-    return validate_empty_field(REPORT_GENDER.get(value))
+def get_gender_as_string(gender: Optional[Gender]) -> str:
+    """Return a report adapted gender."""
+    return get_report_string(REPORT_GENDER.get(gender)) if gender else NA_FIELD
```

Comment:
```suggestion
    return get_report_string(REPORT_GENDER.get(gender))
```
The get_report_string already returns NA_FIELD right?

Reply:
Good point! 👍 For some reason I though that get(None) will raise an error